drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -815,10 +815,8 @@ void RasterizerCanvasGLES3::_render_item(RID p_render_target, const Item *p_item
 				_bind_canvas_texture(texture, current_filter, current_repeat, r_index);
 				if (instance_count == 1) {
 					GLES3::MaterialStorage::get_singleton()->shaders.canvas_shader.version_bind_shader(state.current_shader_version, CanvasShaderGLES3::MODE_ATTRIBUTES);
-				} else if (instance_count > 1) {
-					GLES3::MaterialStorage::get_singleton()->shaders.canvas_shader.version_bind_shader(state.current_shader_version, CanvasShaderGLES3::MODE_INSTANCED);
 				} else {
-					ERR_PRINT("Must have at least one mesh instance to draw mesh");
+					GLES3::MaterialStorage::get_singleton()->shaders.canvas_shader.version_bind_shader(state.current_shader_version, CanvasShaderGLES3::MODE_INSTANCED);
 				}
```

Comment:
Doesn't this change behaviour though?

Reply:
I think `instance_count >= 2` because of break in
https://github.com/godotengine/godot/blob/master/drivers/gles3/rasterizer_canvas_gles3.cpp#L790-L794

Reply:
That line only applies to MultiMesh. I'm not sure why, but I think sometimes Mesh sneak through with an instance count of 0. Also soon we will have particles implemented as well, which will be a third case to handle